### PR TITLE
Fixes for Path of Shadows Book

### PR DESCRIPTION
**Pull request by Diggs232**
Corrections for nightblades and archetypes not functioning properly

### CODE REVIEW

File: data/pathfinder/ascension_games/path_of_shadow/pos_classes.lst [hunk 1]
```diff
@@ -7,49 +7,48 @@ CLASS:Nightblade	HD:8		TYPE:Base.PC	MAXLEVEL:20	SOURCEPAGE:p.6	DEFINE:Nightblade
...
+15	CAST:0,5,5,5,4,3		KNOWN:6,6,6,5,4,4
+16	CAST:0,5,5,5,4,3,1	KNOWN:6,6,6,5,5,4,2
+17	CAST:0,5,5,5,4,4,2	KNOWN:6,6,6,6,5,4,3
+18	CAST:0,5,5,5,5,4,3	KNOWN:6,6,6,6,5,4,4
+19	CAST:0,5,5,5,5,5,4	KNOWN:6,6,6,6,5,5,4
+20	CAST:0,5,5,5,5,5,5	KNOWN:6,6,6,6,6,5,5
+
 ###Block
-1	ABILITY:Class|AUTOMATIC|Nightblade
-CLASS:Druid.MOD	DEFINE:ProhibitSpell_Descriptor_Light|0
```
Comment (LegacyKing) [inline data/pathfinder/ascension_games/path_of_shadow/pos_classes.lst:54]: Why are we deleting this?

Comment (Diggs232) [reply data/pathfinder/ascension_games/path_of_shadow/pos_classes.lst:54]: It was defined as 0 in two files and I completely missed that it should have been defined as 1 in this file. Thank you for catching this

Comment (Diggs232) [reply data/pathfinder/ascension_games/path_of_shadow/pos_classes.lst:54]: to clarify: It is defined in the file pos_abilities_class.lst line 37. 
Should I add it back in here and remove it there? or have it in both places?

Comment (Diggs232) [reply data/pathfinder/ascension_games/path_of_shadow/pos_classes.lst:54]: Fixed

**Pull request merged**